### PR TITLE
Docs.dasharo update - OS versioning removal

### PR DESCRIPTION
**Pull request by JanPrusinowski**
Docs.dasharo update - OS versioning removal

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-stability/C04-power-management.md [hunk 1]
```diff
@@ -442,7 +442,7 @@ Powering on should proceed without issues on each attempt. All basic
 functionality (i.e. builtin keyboard, touchpad, display etc) should keep
 working.
 
-## SPM006.002 2x warmboot cycle (AC) (Windows 11)
+## SPM006.002 2x warmboot cycle (AC)
```
[the remark this answers is not in the document]
Comment (BeataZdunczyk) [reply docs/unified-test-documentation/dasharo-stability/C04-power-management.md:445]: We shouldn't remove all information about the OSs. We should leave information about the OS (Ubuntu/Windows), but remove the versioning.
> Is it always the case? Should we document this in a place where we explain which OSes are tested?

We could check. I'd just leave the OS type in the test name (in parentheses) for now.

[the remark this answers is not in the document]
Comment (JanPrusinowski) [reply docs/unified-test-documentation/dasharo-stability/C04-power-management.md]: changed in: 5dabc6684fd90248cc71882565b562e1bf48f0cb

File: docs/unified-test-documentation/overview.md [hunk 2]
```diff
@@ -39,3 +39,19 @@ be distinguished:
...
+* Debian 11.0
+* Fedora 37
+* FreeBSD 13.2
+* Proxmox VE 7.4
+* OPNsense 23.01
+* pfSense CE 2.6.0
+
+`* unless specified otherwise in the test`
+
+(as of 17 june 2024)
```
Comment (BeataZdunczyk) [inline docs/unified-test-documentation/overview.md:57]:
```suggestion
(as of 17 June 2024)

File: docs/unified-test-documentation/overview.md [hunk 3]
```diff
@@ -39,3 +39,19 @@ be distinguished:
...
+* Ubuntu 22.04
+* Ubuntu Server 22.04 LTS
+* Debian 11.0
+* Fedora 37
+* FreeBSD 13.2
+* Proxmox VE 7.4
+* OPNsense 23.01
+* pfSense CE 2.6.0
+
+`* unless specified otherwise in the test`
```
Comment (BeataZdunczyk) [inline docs/unified-test-documentation/overview.md:55]:
```suggestion
`* Unless specified otherwise in the test`

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

**Review by macpijan — APPROVED** [review]

**Pull request merged**